.github/workflows/build-latest.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     with:
       arch: amd64
       tag: latest-amd64
-      core_ref: v19.13.0
+      core_ref: v19.14.0
```

Comment:
I don't see a git tag for `v19.14.0` in the stellar/stellar-core repository, which is why the CI checks are failing as they can't find the tag either.

Reply:
oops, opened the pr a bit too soon. this is fixed now! (and Makefile is updated as well)

Reply:
@leighmcculloch v19.14.0 is created, and CI passed all tests for this PR